common/test/acceptance/pages/studio/overview.py

```diff
@@ -126,29 +128,42 @@ def in_editable_form(self):
         """
         Return whether this outline item's display name is in its editable form.
         """
+        # pylint: disable=no-member
```

Comment:
Because there were multiple lines in a row that needed it. This is a weird case because the first few classes in this file are intended to be mixins to page objects, so they reference things that a `Page` will have (eg `self.q`) but that a blank `object` will not.